src/main/java/roomescape/reservation/ReservationRepository.java

```diff
@@ -0,0 +1,77 @@
+package roomescape.reservation;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.reservationTime.ReservationTime;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class ReservationRepository {
+	private final JdbcTemplate jdbcTemplate;
+
+	public ReservationRepository(JdbcTemplate jdbcTemplate) {
+		this.jdbcTemplate = jdbcTemplate;
+	}
+
+	public List<Reservation> find() {
+		String sql = "SELECT \n" +
+				"    r.id as reservation_id, \n" +
+				"    r.name as reservation_name, \n" +
+				"    r.date as reservation_date, \n" +
+				"    t.id as time_id, \n" +
+				"    t.start_at as time_start_at \n" +
+				"FROM reservation as r \n" +
+				"inner join reservation_time as t \n" +
+				"on r.time_id = t.id";
+		return jdbcTemplate.query(sql,
+				(rs, rowNum) ->
+						new Reservation(rs.getLong("reservation_id"),
+								rs.getString("reservation_name"),
+								rs.getString("reservation_date"),
+								new ReservationTime(rs.getLong("time_id"), rs.getString("time_start_at")))
+		);
```

Comment:
```suggestion
        return jdbcTemplate.query(sql, (rs, rowNum) ->
            new Reservation(rs.getLong("reservation_id"),
                rs.getString("reservation_name"),
                rs.getString("reservation_date"),
                new ReservationTime(rs.getLong("time_id"), rs.getString("time_start_at")))
        );
```
tab 을 space 형식으로 바꾸고, 더 보기 편하게 줄바꿈을 시도해봐도 괜찮을 것 같아요.
코딩 스타일은 꽤 많은 분들이 Google 의 방식을 사용하기 때문에, [그 방식을 한 번 확인해보고](https://google.github.io/styleguide/javaguide.html) 적용해보는걸 추천드려요!

위 링크에 적힌 컨벤션 내용을 보면 `tab` 을 인덴트 용도로 사용하지 않는다는 부분이 있어요 : 
<img width="1095" alt="Screenshot 2024-06-06 at 8 19 41 PM" src="https://github.com/next-step/spring-roomescape-reservation/assets/48251668/f4a793dc-9ca8-4aa7-8d83-2c109b99dd33">

Reply:
리뷰어는 아니지만~ 코멘트 보다가 괜찮은 주제인 것 같아서 의견 남겨요 ㅎㅎ
https://github.com/spring-io/spring-javaformat
이런 것도 있으니 참고 해보면 좋을 거 같아요~